packetizer.go

```diff
@@ -68,5 +88,27 @@ func (p *packetizer) Packetize(payload []byte, samples uint32) []*Packet {
 	}
 	p.Timestamp += samples
 
+	if p.AbsSendTime != 0 {
+		t := toNtpTime(time.Now()) >> 14
+		//apply http://www.webrtc.org/experiments/rtp-hdrext/abs-send-time
+		packets[len(payloads)-1].Header.Extension = true
+		packets[len(payloads)-1].ExtensionProfile = 0xBEDE
+		packets[len(payloads)-1].ExtensionPayload = []byte{
+			//the first byte is
+			// 0 1 2 3 4 5 6 7
+			//+-+-+-+-+-+-+-+-+
+			//|  ID   |  len  |
+			//+-+-+-+-+-+-+-+-+
+			//per RFC65285
```

Comment:
Do you have a link for the RFC? I couldnt find it

Reply:
the RFC number is a typo, it's 5285. sorry about that.

https://tools.ietf.org/html/rfc5285